### ISSUE TITLE
Translation and contrast issues

### ISSUE THREAD

**Issue by xavierfiechter** [KoalaSat/nostros#337]
![Screenshot_20230215-224652](https://user-images.githubusercontent.com/31884704/219177476-5264c71a-0b52-4e93-aef4-4441f50f4cb7.png)
![Screenshot_20230215-224705](https://user-images.githubusercontent.com/31884704/219177501-81eac599-8a3d-4290-b53c-b7e263824291.png)

A: better contrast needed
B: "Adresse"
C: "Meine Relays" 
D: better contrast needed
E: "Einen Fehler melden"
F: "Kontakte" 

@123vier @KoalaSat

**Comment by 123vier** [comment]
Thank you, I'm taking care of this
Danke

**Comment by 123vier** [comment]
@KoalaSat for the issues in the first picture, the main menu, translation already exists, I compared it with en.json and all items are translated. I only noticed that one line was in a different order and corrected this, but as far as I know this shouldn't make a difference inside this section. If there's a syntax error, I cannot find it.

 "menuItems": {
      "relays": "Relays",
      "contacts": "Kontakte",
      "notConnected": "Nicht verbunden",
      "connectedRelays": "{{number}} verbunden",
      "following": "{{following}} folge ich",
      "followers": "{{followers}} folgen mir",
      "configuration": "Einstellungen",
      "about": "Über Nostros",
      "contacts": "Kontakte",
      "reportBug": "Bug melden",
      "logout": "Abmelden"
    },

**Comment by 123vier** [comment]
same issue with

"myList": "Meine Relays",

it shows the text from the en.json

"myList": "My relays",

**Comment by 123vier** [comment]
@FrontendCrypto can you have a look at the contrast of these items in the screenshot, labeled A and D please?

thanks

**Comment by FrontendCrypto** [comment]
Yes, I think that is a problem. I have designed it differently. Not implemented as in design.

![Screenshot_2023-02-16-10-09-43-093_com.figma.mirror.jpg](https://user-images.githubusercontent.com/4659020/219319669-be2c2809-171b-4366-8b68-c44e430c8987.jpg)

**Comment by xavierfiechter** [comment]
Thanks!
